### ISSUE TITLE
[FEATURE REQUEST] Exit download_data.py dry-run script with error if not using Python 3

### ISSUE THREAD

**Issue by yantosca** [geoschem/geos-chem#389]
@cdholmes suggested:

>The problem for novice users is that the exit error message says something about syntax error in a “print” statement, and nothing about the Python version. I suggest specifying “python3” in the shebang line, as in
``` 
#!/usr/bin/env python3
```
>In addition or instead, I would add a version check after the imports. For example
```
assert sys.version_info.major>=3, "Python 3 required"
```

**Comment by yantosca** [comment]
We will add this into 13.0.0.  Note that run-directory generation in 13.0.0 and later versions will now be handled within the [geoschem/geos-chem(https://github.com/geoschem/geos-chem) repo instead of the [geoschem/geos-chem-unittest](https://github.com/geoschem/geos-chem-unittest) repo, which is being retired.

**Comment by yantosca** [comment]
This issue has now been resolved with commit https://github.com/geoschem/geos-chem/commit/cd1af335f758e9ff244a56a06b38db3b482cc8e8.

Also note corresponding commit https://github.com/geoschem/geos-chem/commit/7ff3f7b318324b56720142ed5801b6131abe23a2 that fixes an bug in the GEOS-Chem 13.0.0 dry-run simulation.
